pennylane/gradients/parameter_shift.py

```diff
@@ -939,7 +1167,7 @@ def processing_fn(results):
     return gradient_tapes, processing_fn
 
 
-# TODO: docstrings
+# TODO: docstrings & mention shots arg

```

Comment:
It can be removed I think.

Reply:
I'd like to mention it explicitly in the docstring beyond documenting the argument.